src/main/java/bio/terra/workspace/service/workspace/flight/CreateGoogleContextFlight.java

```diff
@@ -43,6 +43,7 @@ public CreateGoogleContextFlight(FlightMap inputParameters, Object applicationCo
         new RetryRuleExponentialBackoff(
             INITIAL_INTERVAL_SECONDS, MAX_INTERVAL_SECONDS, MAX_OPERATION_TIME_SECONDS);
     addStep(new SetProjectBillingStep(crl.getCloudBillingClientCow()));
+    addStep(new CreateCustomGcpRolesStep(crl.getIamCow()));
```

Comment:
There was a `throw RetryException` in the step class, so you need a RetryRule here.

Reply:
good catch, thanks!